### ISSUE TITLE
NCAAF Quarterback Position does not Populate

### ISSUE THREAD

**Issue by soySmelliot** [roclark/sportsreference#145]
**Describe the bug**
When pulling a quarterback's information, the position attribute is an empty string. (ex: Adrian Martinez, https://www.sports-reference.com/cfb/players/adrian-martinez-2.html)

**To Reproduce**
```python
>>>am = Player('adrian-martinez-2')
>>>am.position
''
```

**Expected behavior**
For a quarterback, the returned string should be 'QB'. 

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Sportsreference Version 0.4.3

**Comment by roclark** [comment]
Hey @soySmelliot, thanks for bringing this to my attention! Looks like none of the NCAAF player pages contain a position in the career stats field, so all players show as having no position by default. I created a fix for this by pulling the most recent season's position whenever the career position cannot be found. From my testing, this is the same as the position listed in the information panel at the top of each page, so it should be consistent.

I pushed a fix to the repository just a bit ago, but I want to take care of some other un-related tests that are failing at the moment before I merge it. Thanks again for the help!
